### ISSUE TITLE
Gideline select

### ISSUE THREAD

**Issue by SReaver** [avocode/avocode#2183]
---
* **Version**: 3.5.5
* **Web/Desktop**: Desktop
* **Platform**: win32
* **Avocode UID**: 432303
---
```
Error: Failed to execute 'drawImage' on 'CanvasRenderingContext2D': The image argument is a canvas element with a width or height of 0.
    at Object._renderTickCanvas (file:///C:/Users/sreav/AppData/Local/avocode/app-3.5.5/resources/app.asar/dist/avocode.min.js:1:3479115)
    at Object._updateTickCanvas (file:///C:/Users/sreav/AppData/Local/avocode/app-3.5.5/resources/app.asar/dist/avocode.min.js:1:3478771)
    at Object.componentDidMount (file:///C:/Users/sreav/AppData/Local/avocode/app-3.5.5/resources/app.asar/dist/avocode.min.js:1:3478298)
    at CallbackQueue.notifyAll (C:\Users\sreav\AppData\Local\avocode\app-3.5.5\resources\app.asar\node_modules\react-dom\lib\CallbackQueue.js:76:22)
    at ReactReconcileTransaction.close (C:\Users\sreav\AppData\Local\avocode\app-3.5.5\resources\app.asar\node_modules\react-dom\lib\ReactReconcileTransaction.js:80:26)
    at ReactReconcileTransaction.closeAll (C:\Users\sreav\AppData\Local\avocode\app-3.5.5\resources\app.asar\node_modules\react-dom\
```

**Comment by vojtatranta** [comment]
Hi @SReaver ,
we are sorry about this issue. We have already found a solution and it will be released as soon as possible in the next release.

**Comment by vojtatranta** [comment]
We have just released `3.5.6` version. Update your app and this problem should be resolved.
